test/unit/amplitude/test_yaml_canonical.py

```diff
@@ -49,44 +49,42 @@ def equalize_dict(input_dict):
     return output_dict
 
 
-class TestCanonicalAmplitudeGeneratorYAML:
```

Comment:
I was also thinking about removing the class, but I thought I leave it in since the purpose would be to structure tests into categories. Not sure if that makes sense (if we will have more tests coming up here).

Again I would prefer to keep the responsibility of one PR cleaner.

Reply:
The test file name already serves to bundle the tests related to "canonical amplitude generation in YAML"